packages/backend/src/ai.json

```
@@ -43,6 +45,16 @@
       "popularity": 3,
       "license": "?",
       "url": "https://huggingface.co/TheBloke/Llama-2-7B-Chat-GGUF/resolve/main/llama-2-7b-chat.Q5_K_S.gguf"
+    },
+    {
+      "id": "mistral-7b-instruct-v0.1.Q4_K_M",
+      "name": "Mistral-7B-Instruct-v0.1-GGUF",
+      "description": "The Mistral-7B-Instruct-v0.1 Large Language Model (LLM) is a instruct fine-tuned version of the [Mistral-7B-v0.1](https://huggingface.co/mistralai/Mistral-7B-v0.1) generative text model using a variety of publicly available conversation datasets. For full details of this model please read our [release blog post](https://mistral.ai/news/announcing-mistral-7b/)",
+      "hw": "CPU",
+      "registry": "Hugging Face",
+      "popularity": 3,
+      "license": "apache-2.0",
```

```suggestion
      "license": "Apache-2.0",
```

to follow https://spdx.org/licenses/